### ISSUE TITLE
Save file location ignored

### ISSUE THREAD

**Issue by beastie1** [OtterBrowser/otter-browser#1763]
No matter where a file is saved, it always ends up in **~/Downloads**. Even though I don't even have a **~/Downloads** directory, Otter will create one and store the saved file there.

Tested with the default configuration, on a Master build running on FreeBSD 14.0.

**Comment by Emdek** [comment]
@beastie1, QtWebEngine backend?

**Comment by beastie1** [comment]
Yes. Qt5 WebEngine 5.15.16     On Sunday, April 14, 2024 at 09:40:20 PM GMT+3, Michał ***@***.***> wrote:  
 
 


@beastie1, QtWebEngine backend?
